src/addons/build.js

```diff
@@ -174,56 +174,56 @@ export default (XRegExp) => {
             // Named subpattern
             if (subName) {
                 if (!data.hasOwnProperty(subName)) {
-                    throw new ReferenceError('Undefined property ' + $0);
+                    throw new ReferenceError(`Undefined property ${$0}`);
                 }
                 // Named subpattern was wrapped in a capturing group
                 if ($1) {
                     capName = outerCapNames[numOuterCaps];
                     outerCapsMap[++numOuterCaps] = ++numCaps;
                     // If it's a named group, preserve the name. Otherwise, use the subpattern name
                     // as the capture name
-                    intro = '(?<' + (capName || subName) + '>';
+                    intro = `(?<${capName || subName}>`;
                 } else {
                     intro = '(?:';
                 }
                 numPriorCaps = numCaps;
-                return intro + data[subName].pattern.replace(subParts, (match, paren, backref) => {
+                return `${intro + data[subName].pattern.replace(subParts, (match, paren, backref) => {
```

Comment:
This feels really excessive. At a minimum, this should separately interpolate `intro` and the rest of this. But really, this should probably not use a `` ` `` string at all, unless the rest of this is first assigned to a variable.